pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -77,15 +89,29 @@ func (enqueue *enqueueAction) Execute(ssn *framework.Session) {
 	emptyRes := api.EmptyResource()
 	nodesIdleRes := api.EmptyResource()
 	for _, node := range ssn.Nodes {
-		nodesIdleRes.Add(node.Allocatable.Clone().Multi(1.2).Sub(node.Used))
+		nodesIdleRes.Add(node.Allocatable.Clone().Multi(1.2))
+		nodeUsedRes.Add(node.Used)
+	}
+
+	//deltaRes will equal to the bigger resource consumption
+	deltaRes := api.EmptyResource()
+	if pgUsedRes.Less(nodeUsedRes) {
+		deltaRes = nodeUsedRes
+	} else {
+		deltaRes = pgUsedRes
+	}
+	if deltaRes.Less(nodesIdleRes) {
+		nodesIdleRes.Sub(deltaRes)
+	} else {
+		nodesIdleRes = api.EmptyResource()
```

Comment:
Not understand what does this do?

Reply:
use either node used resource or pg used resource that which is greater here. to perform the strict check。